edgexfoundry/go-mod-core-contracts
# feat!: update API version to v3

BREAKING CHNAGE: all EdgeX base API version is changed to v3

closes https://github.com/edgexfoundry/go-mod-core-contracts/issues/774







If your build fails due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/go-mod-core-contracts/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions

1. Build core services and device-virtual from this branch
2. Start EdgeX and verify all services is working correctly

## New Dependency Instructions (If applicable)
